test/unit/services/import_countries_service_test.rb

```diff
@@ -121,6 +121,16 @@ def geonames_response
     end
   end
 
+  test 'falls back to local static file on timeout' do
+    GeonamesLoader.unstub(:load_countries)
+    stub_request(:get, 'http://download.geonames.org/export/dump/countryInfo.txt').to_raise(Net::OpenTimeout.new)
```

Comment:
I'm not sure if having this link hardcoded everywhere is ideal 🤔 
https://github.com/3scale/porta/blob/7ca690acc92c3dea43779a0f600bd9b99613494a/app/services/import_countries_service.rb#L44

It would be better if it was customizable, but as I know that it is more work than desired, a simple improvement would be simply having it inside a constant and always referring to that same constant from all places instead of copy-pasting the hardcoded url everywhere.

Anyway, I know that this is out of the intended scope of the issue so I will approve 😄